### ISSUE TITLE
Error running instadiff-cli

### ISSUE THREAD

**Issue by marceloavilaoliveira** [obalunenko/instadiff-cli#230]
I'm getting the following error running instadiff-cli:

```
INFO[12-01-2023 10:27:24] Exit...                                      
FATA[12-01-2023 10:27:24] Failed to run                                 error="service setup: make client: failed to login: unexpected: Failed to fetch timeline: json: cannot unmarshal string into Go struct field User.feed_items.media_or_ad.user.fbid_v2 of type int64"
```

In the Instagram App > Configuration > Security > Login activity I can see that instadiff-cli logged in.

app_name:     instadiff-cli
version:      v1.13.3
go_version:   go1.19.4
commit:       09ee4fe74642ac76731ee6877899b99503a92579
short_commit: 09ee4fe7
build_date:   2023-01-08T15:04:01Z

**Comment by obalunenko** [comment]
Hi @marceloavilaoliveira ! 
Thank you for your feedback, I'll take a look and try to figure out what went wrong.
Probably it's related to this PR, where I downgraded API version to avoid blocks from Instagram.

**Comment by marceloavilaoliveira** [comment]
Hi @obalunenko , no news about this issue? Thanks a lot.

**Comment by obalunenko** [comment]
Hi @marceloavilaoliveira ! 
It's in progress.

**Comment by obalunenko** [comment]
Hi @marceloavilaoliveira !
Issue should be fixed in [v1.14.0](https://github.com/obalunenko/instadiff-cli/releases/tag/v1.14.0)

**Comment by marceloavilaoliveira** [comment]
I just tested and it worked as expected now. Thanks a lot for your support.
